<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserMainWindow</class>
 <widget class="QMainWindow" name="UserMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>普通用户</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="userLabel">
      <property name="text">
       <string>User:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="BookQueryWidget" name="bookQueryTab">
       <attribute name="title">
        <string>图书查询</string>
       </attribute>
      </widget>
      <widget class="userLoanWidget" name="loanTab">
       <attribute name="title">
        <string>借阅状态</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>退出</string>
    </property>
    <addaction name="exitAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="exitAction">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BookQueryWidget</class>
   <extends>QWidget</extends>
   <header location="global">bookquerywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>userLoanWidget</class>
   <extends>QWidget</extends>
   <header location="global">userloanwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
